backend/src/test/java/de/dontletyoudie/backend/cucumberglue/CucumberMySteps.java

```diff
@@ -0,0 +1,45 @@
+package de.dontletyoudie.backend.cucumberglue;
+
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.cucumber.spring.CucumberContextConfiguration;
+import org.junit.jupiter.api.Assertions;
```

Comment:
```suggestion
import org.assertj.core.api.Assertions;
```

AFAIK is org.assertj.core.api.Assertions a better and more overarching.
But for this project both would work fine I think.